docs/anchoring-assets/anchor-platform/getting_started.mdx

```diff
@@ -237,6 +237,8 @@ services:
       - "5432:5432"
     env_file:
       - ./dev.env
+    volumes:
+      - ./init.sql:/docker-entrypoint-initdb.d/init.sql
```

Comment:
`CREATE DATABASE <name>` will fail if the DB already exists. I just want to confirm that the docker container won't crash if the init script isn't successful.

Reply:
Only running once
https://github.com/docker-library/postgres/blob/master/docker-entrypoint.sh#L312-L338